packages/sources/src/components/accordion/accordion.stories.ts

```diff
@@ -0,0 +1,219 @@
+import { bindTemplate, componentArgs, StorybookParameters } from '../../stories-helpers';
+
+import { AccordionArgs, accordionArgsMapper, accordionArgTypes } from './accordion.args';
+import { Accordion } from './accordion.models';
+
+export interface AccordionParameters<TemplateFnReturnType> {
+  accordionTemplate: (accordionProperties: Accordion) => TemplateFnReturnType;
+}
+
+export function storiesOfAccordion<TemplateFnReturnType>(
+  {
+    module: mainModule,
+    storiesOf,
+    readme
+  }: StorybookParameters,
+  {
+    accordionTemplate
+  }: AccordionParameters<TemplateFnReturnType>
+) {
+  const template = bindTemplate(accordionArgsMapper, accordionTemplate);
+
+  const stories = storiesOf('Accordion', mainModule)
+    .addParameters({
+      docs: {
+        page: readme
+      },
+      argTypes: accordionArgTypes,
+      args: componentArgs<Pick<AccordionArgs, 'variant'>>({
+        variant: 'default'
```

Comment:
De variant is niet `default`: Volgens Accordion model is het `undefined`, `compact` of `conclusion`.